src/PerfView/GuiUtilities/WebBrowser/WebBrowser.xaml.cs

```diff
@@ -13,7 +13,7 @@ namespace PerfView.GuiUtilities
     /// <summary>
     /// Interaction logic for WebBrowserWindow.xaml
     /// </summary>
-    public partial class WebBrowserWindow : WindowBase
+    public partial class WebBrowserWindow : WindowBase, IDisposable
```

Comment:
Sort of a NIT: Since you're not asking consumers to manage the lifetime of this window, I wouldn't implement IDisposable. The pattern makes sense as an implementation, but I wouldn't expose it (the comment in the Closing() event handler is evidence of this weirdness. The window manages its disposable resources within the "control lifecycle".  I'd get rid of the extra Dispose pattern goo and just have the _disposed bool to control operating on _Browser and call _Browser.Dispose conditionally in the Window_Closing